proto/src/main/proto/Rss.proto

```diff
@@ -92,6 +92,7 @@ message GetMemoryShuffleDataRequest {
   int64 lastBlockId = 4;
   int32 readBufferSize = 5;
   int64 timestamp = 6;
+  optional bytes serializedTaskIdsBitmap = 7;
```

Comment:
Filter may be two kinds. expectTaskIdsBitmap, excludeTaskIdsBitmap.

Reply:
I wonder why we need excludeTaskIdsBitmap?

Reply:
> I wonder why we need excludeTaskIdsBitmap?

For multi replicas, it may be better if we have excludeTaskIdsBitmap. This name is not good.

Reply:
But how do we know which tasks need to be excluded?

Reply:
> But how do we know which tasks need to be excluded?

We have processed tasks.

Reply:
But you're right. Maybe expectTaskIdsBitmap is enough. But we should a better name here.

Reply:
real expectTaskIdsBitmap = expectTaskIdsBitmap - distinctTaskId(expectBlockId-processedBlockId)?

Reply:
> real expectTaskIdsBitmap = expectTaskIdsBitmap - distinctTaskId(expectBlockId-processedBlockId)?

Yes,  you're right. If expectTaskIdsBitmap is a little big, excludeTaskIdBitmap is also useful.

Reply:
`serializedTaskIdsBitmap` -> `serializedExpectedTaskIdsBitmap` ?

Reply:
ok.

Reply:
How about expectTaskIdsBitmap = distinctTaskId(expectBlockId-processedBlockId)?

Reply:
Maybe this could be included in another PR, because I wont use the multiple replicas and lack some performance tests.